### DISCUSSION TITLE
来了奥

### DISCUSSION THREAD

**Discussion by Stormlocken** [share121/inter-knot#4661] [灌水]
看看能不能发张图

**Comment by Stormlocken** [comment]
好像没法出去

**Comment by Stormlocken** [comment]
![Uploading mmexport1725337694842.jpg…]()

**Comment by sensei-is-playing-GenShin** [comment]
![Uploading 微信截图_20240908231931.png…]()
好像可以啊

**Reply by sensei-is-playing-GenShin** [reply]
啊，我记得昨天凌晨还是可以的啊
![微信截图_20240908231931](https://github.com/user-attachments/assets/feb7e7cb-991b-4214-ab72-d50bc5ea1e4c)

**Reply by sensei-is-playing-GenShin** [reply]
诶，又好了

**Comment by nzj2004** [comment]
![SAVE_20240906_112332](https://github.com/user-attachments/assets/68853661-f671-43ec-be52-930eaf78a3e3)
经典
